lib/schemas/store-order-recipient.js

```diff
@@ -2,12 +2,15 @@ const yup = require('yup');
 const config = require('../config');
 
 const storeOrderRecipientSchema = yup.object().shape({
-  id: yup.number().required(),
+  id: yup
+    .string()
+    .required()
+    .matches(/^([0-9a-fA-F]{8}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{4}-[0-9a-fA-F]{12})|[0-9]+$/),
```

Comment:
Optional: Can we assign this to constant for readability? Perhaps `UUID_RE`? (`RE` short for regular expression, which provides context at a glance)

Reply:
Question: Did you steal this pattern from somewhere, or is there something particular about RO UUIDs? If the former, let's link to the source in the JSDoc for `UUID_RE`; if the latter, we should test it.

In fact, we should probably test it, regardless 🤔

Reply:
haha yes I stole the pattern! Let me see if I can use the yup matcher for UUID. The reason I did it this way is because I could not find a way to match a number **OR** a UUID string, so I had to cast the ID to a string and run a regex.